src/HydraAuction/Tx/Common.hs

```diff
@@ -294,6 +294,8 @@ scriptUtxos script terms = do
 
 data AutoCreateParams = AutoCreateParams
   { authoredUtxos :: [(SigningKey PaymentKey, UTxO)]
+  , signers :: [SigningKey PaymentKey]
```

Comment:
Can you name it `additionalSigners` or something like that? Cuz currently is not clear that there is an other thing that signs.

Also we can rename `authoredUtxos` into `signedUtxos`.